### ISSUE TITLE
Benchmark various open and closed source models

### ISSUE THREAD

**Issue by lyie28** [lavague-ai/LaVague#274]
A lot of people are interested in using different models so will be great to benchmark asap so people know what kind of performance to expected from various models

**Comment by dhuynh95** [comment]
Yes @HiImMadness will provide TheWave 2.0 and we will bench those
